sdk/storage/Azure.Storage.DataMovement.Blobs/src/BlobDirectoryStorageResourceContainer.cs

```diff
@@ -28,19 +28,19 @@ public class BlobDirectoryStorageResourceContainer : StorageResourceContainer
         private BlobStorageResourceContainerOptions _options;
 
         /// <summary>
-        /// Can produce uri
+        /// Defines whether the storage resource type can produce a SAS URL.
         /// </summary>
         public override ProduceUriType CanProduceUri => ProduceUriType.ProducesUri;
 
         /// <summary>
-        /// Returns the path
+        /// Gets the path of the storage resource.
         /// </summary>
         public override string Path => _directoryPrefix;
 
         internal Uri _uri;
 
         /// <summary>
-        /// Obtains the Uri of the blob directory resource, which means we can list
```

Comment:
Just want to call out if this docstring is accurate?

Reply:
Hm. I'm not sure what I was trying to write here, but let's just keep it simple as "Gets the URL of the storage resource."